src/ex4/server/routes/api.js

```diff
@@ -0,0 +1,48 @@
+// Define your endpoints here (this is your "controller file")
+import { ItemService } from '../services/item_manager.js';
+const item = new ItemService;
+
+async function getAllTasks(req, res) {
+    const data = await item.getAll();
+    if (!data) data = []
+    res.status(200).json(data);
+}
+
+async function pushTask(req, res) {
+    console.log("create task");
+    const response = await item.addTask(req.body);
+    res.status(200).json(response);
+}
+
+async function createNewList(req, res) {
+    console.log("new list");
+    await item.newList(req.body);
+    res.status(200).json({health: 'ok'});
+}
+
+async function removeTask(req, res) {
+    console.log("delete task");
+    const deletedTask = await item.deleteTask(req.body.name);
+    res.status(200).json(deletedTask);
+}
+
+async function deleteAll(req, res) {
+    console.log("clear bord");
+    await item.clearBord();
+    res.status(200).json({health: 'ok'});
+}
+
+async function setTaskUrgency(req, res) {
+    console.log("urgency changed");
+    const changedTask = await item.changeUrency(req.body.name, req.body.urgency, req.body.color);
```

Comment:
```suggestion
    const { name, urgency, color } = req.body;
    const changedTask = await item.changeUrency(name, urgency, color);
```
deconstruct the values from the body.
It's better this way because then in each endpoint you get all the required params on the top of the function, and it makes it easier to understand the flow